src/app/zap-templates/zcl/zcl-with-test-extensions.json

```diff
@@ -180,6 +181,13 @@
             "AdminFabricIndex",
             "AdminVendorId"
         ],
+        "Ambient Context Sensing": [
+            "HoldTimeLimits",
+            "PredictedActivity",
+            "AmbientContextTypeSupported",
+            "AmbientContextType",
+            "ClusterRevision"
```

Comment:
ClusterRevision should be added when the implementation is written that handles it via non-Ember means.  As things stand, it's impossible to tell whether this annotation is correct.